src/components/forms/validator.ts

```diff
@@ -13,14 +13,14 @@ export const simpleMemoize = (fn) => {
   }
 }
 
-export const required = (value) => (value ? undefined : 'Required')
+export const required = (value?: string) => (value && value.trim() !== '' ? undefined : 'Required')
```

Comment:
@mikheevm 
Why a side-effect in a validator? 
Also, it's failing when this validator it's used in a non-text input, like a checkbox.

Reply:
Fixed here: https://github.com/gnosis/safe-react/tree/e7063e69e2a2d1c238a64179015b1e765f998cb3

Reply:
@nicosampler trim returns a new string so no side-effect, good catch with non-string values, will fix it tomorrow